drivers/can/Kconfig.rcar

```diff
@@ -3,12 +3,10 @@
 # Copyright (c) 2021 IoT.bzh
 # SPDX-License-Identifier: Apache-2.0
 
-DT_COMPAT_RENESAS_RCAR_CAN := renesas,rcar-can
-
 config CAN_RCAR
 	bool "Renesas R-Car CAN Driver"
-	depends on SOC_FAMILY_RCAR
```

Comment:
Isn't this one needed anymore?

Reply:
No. It is implicit since non-RCAR devicetree wont have a `renesas,rcar-can` node.